.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,189 @@
+---
+name: Test Windows
+
+on:
+  pull_request:
+  workflow_dispatch:
+  merge_group:
+
+env:
+  QT_QPA_PLATFORM: minimal
+  # Last known good conan version
+  # 2.0.3 has a bug on Windows (conan-io/conan#13606)
+  CONAN_VERSION: 2.0.2
+
+concurrency:
+  group: test-windows-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-windows:
+    name: "Test ${{ matrix.os }}, Qt ${{ matrix.qt-version }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+        qt-version: [5.15.2, 6.2.4]
```

Comment:
Shouldn't we test on 6.5 or 6.6?

Reply:
Iirc I just took the version I found in the Ubuntu Test, where there was 6.2.4. Haven't really thought about what versions to use

Reply:
Yeah, whichever version is used to build on windows/macos are the ones that should be used.
6.2.4 is used in Ubuntu 22.04 because that's what's available in the system repo

Reply:
Looking into the build.yml 6.5.0 is used for Windows and MacOS is not getting built in GitHub Actions for Qt6 (Just for Qt5), so would change it to 6.5.0 in `test-windows.yml` if I understand it correctly, but I wouldn't know if I should change it to something else on MacOS.

Reply:
The universal macOS build uses 6.5.0, so that would be the good version to use :+1: